### PR TITLE
Use prod build for e2e tests

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
The e2e tests

### Description
This PR changes the e2e workflow. It now uses the prod build, because that is what we actually want to test. It could also fix the firefox timeout.

### Steps

- [x] added implementation
- [x] added / updated tests
- [ ] added / updated documentation
- [ ] extended changelog

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1799,7 +1799,7 @@
   dependencies:
     "@types/markdown-it" "*"
 
-"@types/markdown-it@10.0.2":
+"@types/markdown-it@*", "@types/markdown-it@10.0.2":
```
Comment (DerMolly) [inline yarn.lock:1802]: did we want to avoid * imports

File: yarn.lock [hunk 2]
```diff
@@ -4186,7 +4196,7 @@ cyclist@^1.0.1:
   resolved "https://registry.yarnpkg.com/cyclist/-/cyclist-1.0.1.tgz#596e9698fd0c80e12038c2b82d6eb1b35b6224d9"
   integrity sha1-WW6WmP0MgOEgOMK4LW6xs1tiJNk=
 
-cypress@4.12.1:
+cypress@*, cypress@4.12.1:
```
Comment (DerMolly) [inline yarn.lock:4199]: same here

**Review by DerMolly — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (mrdrogdrog) [reply yarn.lock:1802]: We want to avoid separated references. Combined ones are okay and work as expected.

**Review by ErikMichelson — APPROVED** [review]
Seems to work fine, although I'm interested about where these requests (typing h-t-t-p-s in the URL bar) originate from?

image

**Comment by mrdrogdrog** [comment]
> Seems to work fine, although I'm interested about where these requests (typing h-t-t-p-s in the URL bar) originate from?
> 
> image

I saw that too.. And I have NO idea o.ö

**Review by DerMolly — APPROVED** [review]

File: yarn.lock (hunk 2 above)
Comment (mrdrogdrog) [reply yarn.lock:4199]: same here

**Pull request merged**